zh/enable-tidb-cluster-auto-scaling.md

```diff
@@ -0,0 +1,193 @@
+---
+title: 启用 TidbCluster 弹性伸缩
+summary: 介绍如何使用 TidbCluster 的弹性伸缩能力。
+category: how-to
+---
+
+# 启用 TidbCluster 弹性伸缩
+
+在 Kubernetes 平台上，有着基于 CPU 利用率进行负载的原生 API: [Horizontal Pod Autoscaler](https://kubernetes.io/zh/docs/tasks/run-application/horizontal-pod-autoscale/)。与之相应的，在 Operator 1.1 版本及以上，TiDB 集群可以凭借 Kubernetes 平台本身的特性来开启弹性调度的能力。本篇文章将会介绍如何开启并使用 TidbCluster 的弹性伸缩能力。
+
+## 开启弹性伸缩特性
+
+> **警告：**
+>
+> * TidbCluster 弹性伸缩目前仍处于 Alpha 阶段，我们极其不推荐在关键、生产环境开启这个特性
+> * 我们推荐你在测试、内网环境对这个特性进行体验，并反馈相关的建议与问题给我们，帮助我们更好地提高这一特性能力。
+
+开启弹性伸缩特性需要主动开启 Operator 相关配置，默认情况下 Operator 的弹性伸缩特性是关闭的。你可以通过以下方式来开启弹性调度特性:
+
+1. 修改 Operator 的 `values.yaml`
+
+在 `features` 选项中开启 AutoScaling：
+
+    ```yaml
+    features:
+      - AutoScaling=true
+    ```
+
+开启 Operator Webhook 特性:
+
+    ```yaml
+    admissionWebhook:
+      create: true
+      mutation:
+        pods: true
+    ```
+
+2. 安装/更新 TiDB Operator
+
+    修改完 `values.yaml` 文件中上述配置项以后进行 TiDB Operator 部署或者更新。安装与更新 Operator 请参考[在 Kubernetes 上部署 TiDB Operator](deploy-tidb-operator.md)。
+
+## 了解 TidbClusterAutoScaler
+
+我们通过 `TidbClusterAutoScaler` CR 对象来控制 TiDB 集群的弹性伸缩行为，如果你曾经使用过 [Horizontal Pod Autoscaler](https://kubernetes.io/zh/docs/tasks/run-application/horizontal-pod-autoscale/)，那么你一定会对这个概念感到非常熟悉。以下是一个 TiKV 的弹性伸缩例子。
+
+    ```yaml
```

Comment:
No need to indent the code block here, right? @ran-huang

Reply:
Yes. If the code block is part of the text, no need to indent. Only indent code blocks when they are under a list.

Reply:
updated